src/Codeception/Module/Yii2.php

```diff
@@ -386,7 +381,7 @@ public function _after(TestInterface $test): void
             $this->loadedFixtures = [];
         }
 
-        $this->getClient()?->resetApplication();
+        $this->getClient()->resetApplication();
```

Comment:
The return type for `getClient()` includes `null`, so we need optional chaining I think?

Reply:
No, if client is `null` the if statement is false. It specifically checks for implementations of `AbstractBrowser` that are not `Yii2Connector`.
But if `$this->client`  is null nothing is checked.

Reply:
The issue is more in codeception design. The `->client` property can be `null`, although we never expect it.
A clean for us could be to say that we require it to not be null.

Something like this: 
```php
    private function getClient(): \Codeception\Lib\Connector\Yii2
    {
        if (!isset($this->client)) {
            throw new \RuntimeException('Browser not initialized')
        }
        if ($this->client instanceof \Codeception\Lib\Connector\Yii2) {
            throw new \RuntimeException('The Yii2 module must be used with the Yii2 browser client');
        }
        return $this->client;
    }

```
So we fail hard and dont ahve to deal with the nullability everywhere else.